packages/web-fetch/src/index.ts

```diff
@@ -0,0 +1,96 @@
+import {reatomAsync} from '@reatom/async'
+import {Atom, Ctx, isAtom, throwReatomError} from '@reatom/core'
+
+export interface ReatomFetchOptions<O, I> {
+	fetch?: (url: string, init: RequestInit) => Promise<Response>
+	init?: Omit<RequestInit, 'method' | 'signal' | 'body' | 'headers'>
+	auth?: MaybeAtom<string>
+	headers?: MaybeAtom<HeadersInit>
+	baseUrl?: string
+	method?: string
+	validate?: (o: O) => boolean | string
+	parseResponse?: (response: Response) => Promise<O>
+	serializeBody?: (i: I, headers: Headers) => BodyInit
+	validateStatus?: (status: number) => boolean | string
+}
+
+type UrlSearchParamsInit = ConstructorParameters<typeof URLSearchParams>[0]
+
+type MaybeAtom<T> = Atom<T> | T
+
+const defaults: ReatomFetchOptions<unknown, unknown> = {
+	init: {},
+	fetch,
```

Comment:
This could fail the whole application if a polifil is not loaded

```suggestion
	fetch: typeof fetch !== 'undefined' ? fetch : fetch as unknown as typeof fetch,
```

Reply:
You have node 16 and want to initiate `reatomFetch` with ponifyl explicitly pass custom `fetch` implementation to `createRetomFetch`.
<img width="278" alt="CleanShot 2023-04-16 at 05 44 40@2x" src="https://user-images.githubusercontent.com/27290320/232263389-7267e9de-ffe2-4f49-b456-f18ef2e9bc36.png">